### ISSUE TITLE
git remote -v

### ISSUE THREAD

**Issue by pink-mist** [LorenzoTa/step-by-step-tutorial-on-perl-module-creation-with-tests-and-git#5]
You write that ```git remote -v``` means to verify the git remote, but the ```-v``` only stands for ```verbose``` output. I'd suggest rewording that part.

**Comment by LorenzoTa** [comment]
adopted:
https://github.com/LorenzoTa/step-by-step-tutorial-on-perl-module-creation-with-tests-and-git/commit/49bfc74662830e616e05cf5eb97f0d275117693d
and
https://github.com/LorenzoTa/step-by-step-tutorial-on-perl-module-creation-with-tests-and-git/commit/c55cb70db00b442ddb6c5a252de3da73d4f15bcb
